packages/ariakit/src/checkbox/__examples__/checkbox/test.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  click,
+  getByLabelText,
+  getByRole,
+  press,
+  render,
+} from "ariakit-test-utils";
+import Example from ".";
+
+test("render checkbox", async () => {
+  render(<Example />);
+  expect(getByRole("checkbox")).toMatchInlineSnapshot(`
```

Comment:
Maybe we should get the wrapping label here to get the whole markup. Does `getByLabelText()` return the label element as well?

Reply:
getByLabelText  returns only the input field. The alternative would be to make a snapshot using the container element.